source/includes/code-examples/FindManySync.cs

```diff
@@ -0,0 +1,97 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+
+namespace CSharpExamples.UsageExamples;
+
+public class FindMany
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
+
+    public static void Main(string[] args)
+    {
+        Setup();
+
+        // Find multiple documents using builders
+        Console.WriteLine("Finding documents with builders...:");
+        FindMultipleRestaurantsBuilderSync();
+
+        // Extra space for console readability 
+        Console.WriteLine();
+
+        // Find multiple documents using LINQ
+        Console.WriteLine("Finding documents with LINQ...:");
+        FindMultipleRestaurantsLINQSync();
+
+        Console.WriteLine();
+
+        // Find All restaurants
+        Console.WriteLine("Finding all documents...:");
+        FindAllRestaurantsSync();
+    }
+
+    public static void FindMultipleRestaurantsBuilderSync()
```

Comment:
If we're going to be making these complete files publicly visible to readers, we should probably follow best style practices. That means all methods besides `Main` should be `private`.